cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -213,25 +213,47 @@ def capacity_reservation_matches_instance(capacity_reservation: Dict, instance_t
     ] == AWSApi.instance().ec2.get_subnet_avail_zone(subnet)
 
 
+def capacity_reservation_resource_group_is_service_linked_group(capacity_reservation_resource_group_arn: str):
+    try:
+        group_config = AWSApi.instance().resource_groups.get_group_configuration(
+            group=capacity_reservation_resource_group_arn
+        )
+        is_cr_pool = False
+        for config in group_config["GroupConfiguration"]["Configuration"]:
+            if "CapacityReservationPool" in config["Type"]:
+                is_cr_pool = True
+        return is_cr_pool
```

Comment:
According to https://docs.aws.amazon.com/ARG/latest/APIReference/about-slg.html, it seems `group_config["GroupConfiguration"]["Configuration"]` is a list of a single item :)
Anyway, we can keep the code as it is to be robust

Reply:
I'm not sure if that's always the case though

```
An item in a group configuration. A group service configuration can have one or more items. For details about group service configuration syntax, see [Service configurations for resource groups](https://docs.aws.amazon.com/ARG/latest/APIReference/about-slg.html).
```